person.rb

```diff
@@ -1,14 +1,21 @@
-class Person
+require './nameable'
+
```

Comment:
- [ ] Kindly add the require `'./capitalize_decorator'` and `'./trimmer_decorator'` class to be avaible in person file. And be sure to run the example on the requirements without errors